tests/unit/Core/ResponseTest.php

```diff
@@ -110,7 +110,7 @@ public function testToArrayWithError() {
         $response = $this->makeResponse('Member.Profile.Get', $this->errorResponseXML);
 
         //Execute and Verify
-        $this->tester->expectException(YourMembershipException::class, function() use ($response)
+        $this->tester->expectException(YourMembershipApiException::class, function() use ($response)
```

Comment:
I haven't used this syntax for exceptions before. I usually use the @expectException PHPUnit annotation on the test method. It may be less code. Syntax is like

@expectException YourMembershipApiException

and then in the test, you'd just call:

$a = $response->toArray()



I'd lean towards that, but it's a matter of preference if it's working.